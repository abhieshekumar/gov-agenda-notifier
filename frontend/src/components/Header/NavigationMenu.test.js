import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import NavigationMenu from './NavigationMenu';

import renderer from 'react-test-renderer';

it('NavigationMenu renders correctly', () => {
    const tree = renderer.create(
        <Router>
            <NavigationMenu />
        </Router>).toJSON();
    expect(tree).toMatchSnapshot();
});
