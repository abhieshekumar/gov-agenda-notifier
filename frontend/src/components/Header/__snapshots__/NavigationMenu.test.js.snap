// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationMenu renders correctly 1`] = `
<div
  className="navigation-menu responsive-padding hide"
>
  <span
    className="hamburger-menu"
    style={
      Object {
        "color": "white",
      }
    }
  >
    <i
      className="fas fa-bars fa-2x"
    />
  </span>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
  </ul>
</div>
`;
